FFmpeg.AutoGen/FFmpeg.cs

```diff
@@ -51,9 +52,78 @@ static ffmpeg()
         public static string RootPath { get; set; } = AppDomain.CurrentDomain.BaseDirectory;
 
         public static GetOrLoadLibrary GetOrLoadLibrary { get; set; }
+        
+        /// <summary>
+        ///     Tries to load the native libraries from the set root path. <br/>
+        ///     You can specify which libraries need to be loaded with LibraryFlags.
+        ///     It will try to load all librares by default. <br/>
+        ///     Ideally, you would want to only call this function once, before doing anything with FFmpeg.
+        ///     If you try to do that later, it might unload all of your already loaded libraries and fail to provide them again.
+        /// </summary>
+        /// <returns>Whether it succeeded in loading all the requested libraries.</returns>
+        public static bool CanLoadLibraries(LibraryFlags libraries = LibraryFlags.All, string path = "")
+        {
+            var validated = new List<string>();
+            return libraries.ToStrings().All((lib) => canLoadLibrary(lib, path, validated));
+        }
+
+        // Note: recurses in a very similar way to the LoadLibrary() method.
+        private static bool canLoadLibrary(string lib, string path, List<string> validated)
+        {
+            Console.WriteLine($"Checking if {lib} can be loaded from {path}");
```

Comment:
I would like to have this kind of tracing when something goes wrong, but I don't want that noise in my application if everything is ok.

Reply:
Yes, sorry, that was a statement that I wrote for debugging and forgot to remove.